negotiation/contract.negotiation.protocol.md

```diff
@@ -100,6 +100,9 @@ The `ContractRequestMessage` is sent by a consumer to initiate a contract negoti
 
 The `ContractOfferMessage` is sent by a provider to initiate a contract negotiation.
```

Comment:
We probably should also update this, since the conract offer is not only sent to initiate but also after the consumer requested a contract.
Similarly for the ContractRequestMessage which can also be send during the flow and not only in initiation.
But this isn't directly part of this PR

Reply:
Addressed in #151